travis/test_measures.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+cd fhir-patient-generator
+
+make MEASURE_DIR=EXM_130
```

Comment:
Should this line be here, since you said in the PR description "I have since removed the make call" ?

Reply:
My bad failed to commit that. Updated with a force push